packages/sqlite/package.json

```diff
@@ -56,6 +56,8 @@
 	},
 	"homepage": "https://github.com/jaredwray/keyv",
 	"dependencies": {
+		"@types/better-sqlite3": "^7.6.3",
```

Comment:
These lines should be removed as we do not have a dependency on this

Reply:
Fixed